subsys/partition_manager/Kconfig

```diff
@@ -6,6 +6,20 @@
 
 menu "Partition Manager"
 
+config PARTITION_MANAGER_ENABLED
+	# Hidden option to indicate to kconfig partition manager is enabled
```

Comment:
This can't be hidden since the option is either set through the overlay (ie prj.conf) or command line directly (whitelisted by multi_image.cmake)